examples/observability/Makefile

```diff
@@ -1,9 +1,18 @@
 JAEGER_VERSION ?= v1.38.0
 TOOLKIT_NAMESPACE ?= keptn-lifecycle-toolkit-system
 PODTATO_NAMESPACE ?= podtato-kubectl
+GRAFANA_PORT_FORWARD ?= 3000
 
 .PHONY: install
 install:undeploy-podtatohead
+	@echo "-----------------------------------"
+	@echo "Create Namespace and install Keptn-lifecycle-toolkit"
+	@echo "-----------------------------------"
+	kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.10.0/cert-manager.yaml
+	kubectl wait --for=condition=available deployment/cert-manager -n cert-manager --timeout=300s
+	kubectl apply -f https://github.com/keptn/lifecycle-toolkit/releases/download/v0.4.0/manifest.yaml
```

Comment:
you could add the release please annotation here to update the version automatically